src/integrations/GA4/browser.js

```diff
@@ -160,7 +161,11 @@ export default class GA4 {
     if (checkRequiredParameters && !hasRequiredParameters(parameters, eventMappingObj)) {
       throw Error('Payload must have required parameters..');
     }
-    window.gtag("event", event, parameters);
+    const params = { ...parameters };
+    if (this.addSendToParameter) {
+      params.send_to = this.measurementId;
+    }
+    window.gtag('event', event, params);
```

Comment:
```suggestion
    if (this.addSendToParameter) {
      parameters.send_to = this.measurementId;
    }
    window.gtag('event', event, parameters);
```
Do we need to create another variable?

Reply:
ESlint error because we are modifying functional argument. So i created a new variable